### PR TITLE
fix(module:tree): amend documentation of check/uncheck all

### PR DESCRIPTION
**Pull request by lukblazewicz**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

Misleading description of the methods.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | amend code-documentation of check/uncheck all |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by lukblazewicz** [comment]
I think the next step should be renaming both methods to CheckAll() and UncheckAll() respectively but obviously, as a breaking change, it should be planned for the next major release. What do you think?

File: components/tree/Tree.razor.cs [hunk 1]
```diff
@@ -324,7 +324,9 @@ public void CheckedAll()
             }
         }
 
-        // Decheck all of the checked nodes
+        /// <summary>
+        /// Unchecks all nodes.
+        /// </summary>
         public void DecheckedAll()
```
Comment (ElderJames) [inline components/tree/Tree.razor.cs:330]: Should we rename this method?

Comment (lukblazewicz) [reply components/tree/Tree.razor.cs:330]: I believe so:
https://github.com/ant-design-blazor/ant-design-blazor/pull/1790#issuecomment-888817711

**Comment by ElderJames** [comment]
> I think the next step should be renaming both methods to CheckAll() and UncheckAll() respectively but obviously, as a breaking change, it should be planned for the next major release. What do you think?

I think we can bring all to `feature` branch and will ship in next major release.

**Comment by lukblazewicz** [comment]
Okay, I'll then abandon this PR and include the abovementioned changes along with methods renaming in a separate PR to the feature branch.

**Pull request closed without merging**